lib/verdict/storage/redis_storage.rb

```diff
@@ -26,6 +26,10 @@ def remove(scope, key)
         raise Verdict::StorageError, "Redis error: #{e.message}"
       end
 
+      def cleanup(scope)
+        redis.del("#{@key_prefix}#{scope}")
```

Comment:
Nitpick, but it might be nice to start extracting `"#{@key_prefix}#{scope}"` into its own private method, this is called a bunch of times in this file.

Reply:
I thought that too but since `scope` is injected I decided against it. I'll add something to make it nicer.